index.js

```diff
@@ -47,8 +51,8 @@ function update(baton) {
       replace({
         files: path.resolve(baton.dir, filename),
         // eslint-disable-next-line
-        replace: /\[\/\/\]: contributor-faces(?:(?:\n.*)+\[\/\/\]: contributor-faces)?/,
-        with: `[//]: contributor-faces\n${baton.html}\n[//]: contributor-faces`
+        from: /\[\/\/\]: contributor-faces(?:(?:\n.*)+\[\/\/\]: contributor-faces)?/,
+        to: `[//]: contributor-faces\n${baton.html}\n[//]: contributor-faces`
```

Comment:
Just to make sure, `from` and `to` are aliases to `replace` and `with`?

Reply:
replace and with have been deprecated and it is advised to use their replacements from and to instead.